sdk/compiler/damlc/daml-ide/src/DA/Daml/LanguageServer/SplitGotoDefinition.hs

```diff
@@ -206,7 +206,7 @@ gotoDefinitionByName ideState params = do
     projectConfig <- liftIO $ readProjectConfig (ProjectPath root)
     config <- except $ first (Just . show) $ parseProjectConfig projectConfig
 
-    srcFiles <- maybeTToExceptT "Failed to get source files" $ getDamlFiles $ root </> pSrc config
+    srcFiles <- maybeTToExceptT "Failed to get source files" $ getDamlFiles $ normalise $ root </> pSrc config
```

Comment:
~~Docs for `https://hackage.haskell.org/package/filepath-1.5.3.0/docs/System-FilePath.html#v:normalise` suggest that it doesn't change "."~~

I see, it'll normalize "." </> "myfile" to "myfile"